.travis.yml

```diff
@@ -25,10 +29,19 @@ install:
 
 before_script:
   # Install Varnish
-  - curl http://repo.varnish-cache.org/debian/GPG-key.txt | sudo apt-key add -
-  - echo "deb http://repo.varnish-cache.org/ubuntu/ precise varnish-${VARNISH_VERSION}" | sudo tee -a /etc/apt/sources.list
+  - |
+    if [ "$VARNISH_VERSION" = "5.0.0" ]; then
+      wget https://repo.varnish-cache.org/pkg/${VARNISH_VERSION}/varnish_${VARNISH_VERSION}-1_amd64.deb
+      sudo dpkg -i varnish_${VARNISH_VERSION}-1_amd64.deb
+      sudo apt-get -f install
+    else
+      curl http://repo.varnish-cache.org/debian/GPG-key.txt | sudo apt-key add -
+      echo "deb http://repo.varnish-cache.org/ubuntu/ precise varnish-${VARNISH_VERSION}" | sudo tee -a /etc/apt/sources.list
+      sudo apt-get update -qq
+      sudo apt-get install -qq varnish
+    fi
   - sudo apt-get update -qq
```

Comment:
can we move this to the if case before the `apt-get -f install` line, to not do it twice when having an older varnish?